client/src/components/AddGuests.jsx

```diff
@@ -5,24 +5,30 @@ import GuestList from './GuestList.jsx';
 
 const AddGuests = ({ setTopLevelState, guests, deleteGuest }) => {
   const [name, setName] = useState('');
+  const [duplicateName, setDuplicateName] = useState(false);
+  const [missingName, setMissingName] = useState(false);
```

Comment:
Instead of doing it this way, you could create a `nameError` useState variable and just put the error message you want in dynamically.
```suggestion
  const [nameError, setNameError] = useState('');
```